app/views/collection/show.html.slim

```diff
@@ -64,7 +64,7 @@
       =link_to(user_profile_path(@collection.owner), class: 'userpic')
         =gravatar_image_tag @collection.owner.email, :alt => @collection.owner.display_name
       .username
-        small Owned by
+        small Project by
```

Comment:
This needs a translation, something like: t('.Project by') (make sure the capitalization is write in the message file.)